l10n_it_mis_reports_pl_bs/data/mis_report_pl.xml

```diff
@@ -0,0 +1,1018 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015-2020 Dinamiche Aziendali srl
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data noupdate="0">
+        <record model="mis.report" id="mis_report_pl">
+            <field name="name">CEE income statement balance sheet</field>
```

Comment:
Ho visto ora che anche che in questo file anche in un altro punto viene usato "Income Statement" per "Conto economico" al posto di "Profit and Loss". 
Quindi in https://github.com/OCA/l10n-italy/pull/1640#discussion_r479988259 per uniformità forse è meglio usare:

```suggestion
            <field name="name">EEC Financial Statements - Income Statement</field>
```

Reply:
Ho usato "Profit and Loss" perchè più usato in Odoo e anche in generale mi pare

Reply:
> Ho usato "Profit and Loss" perchè più usato in Odoo e anche in generale mi pare

Meglio direi, così c'è uniformità in tutto il modulo. :+1: